recipe/conda_build_config.yaml

```diff
@@ -1,19 +1,6 @@
-# cuda 9.2, gcc 7.3
-# cuda 10.0, gcc 7.3
-c_compiler_version:
-  - 7.3  # [linux64]
-  - 10   # [(osx and x86_64) or aarch64]
-  - 12   # [osx and arm64]
-cxx_compiler_version:
-  - 7.3  # [linux64]
-  - 10   # [(osx and x86_64) or aarch64]
-  - 12   # [osx and arm64]
-cudatoolkit:
-  - 10.0
```

Comment:
Also may want to keep these lines (see my comment about possibly building gpu variants in the future).

Reply:
I've reverted it but still comment out